pypit_files/wht_isis_blue_long_r300b_d5300.pypit

```diff
@@ -1,34 +1,41 @@
 # This is a comment line
 
-# Change the default settings
-run ncpus 1
-run spectrograph wht_isis_blue
-output verbosity 2
-output overwrite True
-output sorted isis_blue_R300B_d5300
+# Parameters
+[rdx]
+    spectrograph = wht_isis_blue
+    sortroot = isis_blue_R300B_d5300
 
-# reduce
-reduce overscan method median
-trace slits tilts method spca
-trace slits tilts params 1,1,1
-reduce skysub method bspline
-pixelflat combine method median
-pixelflat combine reject level [10.0,10.0]
-#pixelflat norm recnorm False
-arc calibrate IDpixels 871.062,901.7719,1930.0048,2451.620,3364.25658,3886.125
-arc calibrate IDwaves 3248.4769,3274.905,4159.763,4610.656,5402.0634,5854.110
-science extraction maxnumber 3
+[calibrations]
+    [[biasframe]]
+        number = 2
+    [[pixelflatframe]]
+        number = 3
+    [[traceframe]]
+        number = 3
+    [[wavelengths]]
+        IDpixels  = 871.062,901.7719,1930.0048,2451.620,3364.25658,3886.125
```

Comment:
Is this still needed, even with the `general` algorithm in arclines?

Reply:
Good question.  Another one for @profxj.

Reply:
I fear it is as I have yet to do CuNe properly.